### ISSUE TITLE
Incorrect figure on word2vec tutorial

### ISSUE THREAD

**Issue by gqqnbig** [tensorflow/tensorflow#50104]
## URL(s) with the issue:

https://www.tensorflow.org/tutorials/text/word2vec#summary

### Clear description

The summary figure on https://www.tensorflow.org/tutorials/text/word2vec#summary seems incorrect.

The figure shows the index of shimmered is 7 while the code above says the index is 5. 

The red part of the figure has word 'temperature' and 'code', which didn't appear in the input sentence at all. 

The figure also has indexes like 784 and 589. I don't know what's going on here.

As a result, the figure becomes confusing for learners.

Please fix the figure.

### Submit a pull request?

No

**Comment by aavishkarmishra** [comment]
@MarkDaoust  I would like to work on this issue.
Is there any software or website that can help me for correcting this image ?

**Comment by MarkDaoust** [comment]
1. Are the indices in the code deterministic? 
2. It may be easier to update the code to match the images.
3. Usually for diagrams like this I recommend using google-drawings. 

The original author for that tutorial was @enigmaeth (Afroz).

Afroz, could you share a public link to those drawing files so @aavishkarmishra might copy and fix them?

**Comment by Saduf2019** [comment]
@gqqnbig 
Is this still an issue

**Comment by gqqnbig** [comment]
My description still holds true and I didn't see any documentation changes or code changes. I suspect it is still an issue.

**Comment by Yashashree304** [comment]
Actually here the words 'temperature' and 'code' didn't appear in the input sentence but they are some other words used from the vocabulary (A dictionary of all the words in a corpus). Similarly, the indexes in figures like 784 and 598, are indices of some other words from the vocabulary. Hence, this part of the figure is correct. However, this seems confusing for the first time, hence I have added a note to explain the same below the picture. Though I agree that, index of the word shimmered should be 4 instead of 7, and the index of wide should be 2 instead of 7 in red boxes. Hence, I have corrected the same.
